### DISCUSSION TITLE
MY GITHUB PROFILE HAS BEEN HACKED

### DISCUSSION THREAD

**Discussion by ghost** [community/community#131923] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My account has been hacked two months ago, my email has been removed ( [chadyachraf@outlook.com](mailto:chadyachraf@outlook.com) ) and password has changed..
I applied in the form two months ago right after I have been hacked, and a ticket has been created, but without any response ever since.
Here are my credentials :
UserName: Chady00
Email: [chadyachraf@outlook.com](mailto:chadyachraf@outlook.com)

Please help me. I can't access my account anymore.
I am really disappointed at how Github support is not able to assist me although this is an urgent concern.

**Comment by ebndev** [comment]
Hi @Chady001,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
